kymatio/scattering2d/frontend/tensorflow_frontend.py

```diff
@@ -0,0 +1,38 @@
+__all__ = ['Scattering2DTensorflow']
+
+import tensorflow as tf
+
+from ...frontend.tensorflow_frontend import ScatteringTensorFlow
+from kymatio.scattering2d.core.scattering2d import scattering2d
+from .base_frontend import ScatteringBase2D
+
+class ScatteringTensorFlow2D(ScatteringTensorFlow, ScatteringBase2D):
+    def __init__(self, J, shape, L=8, max_order=2, pre_pad=False, backend='tensorflow', name='Scattering2D'):
+        ScatteringTensorFlow.__init__(self, name)
+        ScatteringBase2D.__init__(self, J, shape, L, max_order, pre_pad, backend)
+        ScatteringBase2D._instantiate_backend(self, 'kymatio.scattering2d.backend.')
+        ScatteringBase2D.build(self)
+        ScatteringBase2D.create_filters(self)
+
+    def scattering(self, input):
```

Comment:
We should add some warning checks here like the torch/numpy frontends.

Reply:
I'm not sure it does apply here to a similar extent: tf has no "lowlevel" tensor contrary to torch!

Reply:
> tf has no "lowlevel" tensor contrary to torch!

Could you expand on this a bit?

Reply:
So it seems there are two modes, one which is similar to `torch`, namely the `eager mode` and one for which you need to declare some `tf.placeholder`. I never really used `tf`, thus I'm not sure what is the most standard way to develop. We'd need some tf users' feedbacks to improve this part of the software

Reply:
thoughts?

Reply:
I can ask one of my research group mates, however they haven't used TF much in a while. Everyone in my lab uses pytorch.

Reply:
would be great. It could be done later

Reply:
Ok.